easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.1-intel-2018b-PathCV.eb

```diff
@@ -0,0 +1,54 @@
+# by Ward Poelmans <wpoely86@gmail.com>
+
+easyblock = 'ConfigureMake'
+
+name = 'PLUMED'
+version = '2.5.1'
+versionsuffix = '-PathCV'
+
+homepage = 'http://www.plumed-code.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+ works together with some of the most popular molecular dynamics engines. Free energy calculations can be
+ performed as a function of many order parameters with a particular  focus on biological problems, using
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = [
+    'https://github.com/plumed/plumed2/releases/download/v%(version)s/',
+    'http://www.acmm.nl/ensing/software/',
+]
+sources = [SOURCE_TGZ]
+patches = [('PathCV.cpp', 'src/function/')]
```

Comment:
Where does PathCV.cpp come from?

Reply:
@akesandgren http://www.acmm.nl/ensing/software/

Reply:
Ah, didn't know patches could be downloaded too. (Or have forgotten...)